Description: Rename ATLAS incomplete liblapack.a to liblapack_atlas.a
 The liblapack.a created by ATLAS build system is incomplete: it only contains
 a small subset of the full LAPACK API (only those optimized by ATLAS).
 Hence rename this library to liblapack_atlas.a to avoid a name conflict with
 the full liblapack.a.
 In particular, note that the "-Ss flapack" option of the configure script does
 not work as expected as of ATLAS 3.10.3.
Author: <david@evans>
Forwarded: not-needed
Last-Update: 2017-08-08
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/CONFIG/src/SpewMakeInc.c
+++ b/CONFIG/src/SpewMakeInc.c
@@ -615,7 +615,7 @@ int main(int nargs, char **args)
    fprintf(fpout, "   ATLASlib = $(LIBdir)/libatlas.a\n");
    fprintf(fpout, "   CBLASlib = $(LIBdir)/libcblas.a\n");
    fprintf(fpout, "   F77BLASlib = $(LIBdir)/libf77blas.a\n");
-   fprintf(fpout, "   LAPACKlib = $(LIBdir)/liblapack.a\n");
+   fprintf(fpout, "   LAPACKlib = $(LIBdir)/liblapack_atlas.a\n");
    if (THREADS)
    {
       fprintf(fpout, "   PTCBLASlib = $(LIBdir)/libptcblas.a\n");
--- a/Make.top
+++ b/Make.top
@@ -657,8 +657,8 @@ install_inc:
 install_lib:
 	cp $(LIBdir)/libatlas.a $(INSTdir)/.
 	cp $(LIBdir)/libcblas.a $(INSTdir)/.
-	cp $(LIBdir)/liblapack.a $(INSTdir)/.
-	chmod 0644 $(INSTdir)/libatlas.a  $(INSTdir)/liblapack.a \
+	cp $(LIBdir)/liblapack_atlas.a $(INSTdir)/.
+	chmod 0644 $(INSTdir)/libatlas.a  $(INSTdir)/liblapack_atlas.a \
                    $(INSTdir)/libcblas.a 
 	- cp $(LIBdir)/libf77blas.a $(INSTdir)/.
 	- chmod 0644 $(INSTdir)/libf77blas.a
--- a/makes/Make.lib
+++ b/makes/Make.lib
@@ -4,10 +4,10 @@ mySRCdir = $(SRCdir)/lib
 #
 # override with libatlas.so only when atlas is built to one lib
 #
-DYNlibs = liblapack.so libf77blas.so libcblas.so libatlas.so 
-PTDYNlibs = liblapack.so libptf77blas.so libptcblas.so libatlas.so 
-CDYNlibs = liblapack.so libcblas.so libatlas.so 
-CPTDYNlibs = liblapack.so libptcblas.so libatlas.so 
+DYNlibs = liblapack_atlas.so libf77blas.so libcblas.so libatlas.so 
+PTDYNlibs = liblapack_atlas.so libptf77blas.so libptcblas.so libatlas.so 
+CDYNlibs = liblapack_atlas.so libcblas.so libatlas.so 
+CPTDYNlibs = liblapack_atlas.so libptcblas.so libatlas.so 
 
 VER=3.10.3
 tmpd = RCW_tMp
@@ -26,7 +26,7 @@ $(tarnam).tar.bz2 :
 	cp $(LIBdir)/libatlas.a $(ARCH)/lib/.
 	cp $(LIBdir)/libf77blas.a $(ARCH)/lib/.
 	cp $(LIBdir)/libcblas.a $(ARCH)/lib/.
-	cp $(LIBdir)/liblapack.a $(ARCH)/lib/.
+	cp $(LIBdir)/liblapack_atlas.a $(ARCH)/lib/.
 	- cp $(LIBdir)/libptcblas.a $(ARCH)/lib/.
 	- cp $(LIBdir)/libptf77blas.a $(ARCH)/lib/.
 	$(TAR) cf $(tarnam).tar $(ARCH)
@@ -184,7 +184,7 @@ fat_ptshared :
                 LIBINSTdir="$(LIBINSTdir)"
 fat_shared :                                # serial target
 	$(MAKE) TRYALL outso=libsatlas.so \
-                libas="liblapack.a libf77blas.a libcblas.a libatlas.a" \
+                libas="liblapack_atlas.a libf77blas.a libcblas.a libatlas.a" \
                 LIBINSTdir="$(LIBINSTdir)"
 #
 # Builds shared lib, not include fortran codes from LAPACK
@@ -198,10 +198,10 @@ fat_cshared : libclapack.a
                 libas="libclapack.a libcblas.a libatlas.a" \
                 LIBINSTdir="$(LIBINSTdir)"
 
-libclapack.a : liblapack.a
+libclapack.a : liblapack_atlas.a
 	rm -rf clapack libclapack.a
 	mkdir clapack
-	cd clapack ; ar x ../liblapack.a
+	cd clapack ; ar x ../liblapack_atlas.a
 	rm -f clapack/*f77wrap* clapack/*C2F*
 	ar r libclapack.a clapack/ATL_* clapack/clapack_*
 	rm -rf clapack
@@ -229,7 +229,7 @@ tdlls:                          # thread
                 LIBINSTdir="$(LIBINSTdir)"
 sdlls:                          # serial target
 	$(MAKE) TRYALL_WIN outso=libsatlas.dll outdef=libsatlas.def \
-                libas="liblapack.a libf77blas.a libcblas.a libatlas.a" \
+                libas="liblapack_atlas.a libf77blas.a libcblas.a libatlas.a" \
                 LIBINSTdir="$(LIBINSTdir)"
 cdlls: ctdlls csdlls
 ctdlls: libptclapack.a          # threaded target
@@ -246,7 +246,7 @@ csdlls: libclapack.a            # serial
 #  =======================================================================
 dylib :
 	rm -rf $(tmpd) ; mkdir $(tmpd)
-	cd $(tmpd) ; ar x ../liblapack.a 
+	cd $(tmpd) ; ar x ../liblapack_atlas.a 
 	cd $(tmpd) ; ar x ../libf77blas.a
 	cd $(tmpd) ; ar x ../libcblas.a 
 	cd $(tmpd) ; ar x ../libatlas.a
@@ -283,9 +283,9 @@ ptcdylib : libptclapack.a
         -compatibility_version $(VER) *.o $(LIBS)
 	rm -rf $(tmpd)
 
-libclapack.dylib : libcblas.dylib libatlas.dylib liblapack.a
+libclapack.dylib : libcblas.dylib libatlas.dylib liblapack_atlas.a
 	rm -rf $(tmpd) ; mkdir $(tmpd)
-	cd $(tmpd) ; ar x ../liblapack.a
+	cd $(tmpd) ; ar x ../liblapack_atlas.a
 	rm -f $(tmpd)/*C2F $(tmpd)/*f77wrap*
 	cd $(tmpd) ; libtool -dynamic -o ../libclapack.dylib \
            -install_name $(LIBINSTdir)/libclapack.dylib \
